Resources/public/templates/subitemlist.hbt

```diff
@@ -42,7 +42,23 @@
                         <td>{{#if hidden}}Hidden{{else}}{{#if invisible}}Hidden by superior{{else}}Visible{{/if}}{{/if}}</td>
                         <td>{{contentInfo.publishedDate}}</td>
                         <td>{{contentInfo.lastModificationDate}}</td>
-                        <td>{{priority}}</td>
+                        <td class="ez-subitem-priority-cell" id="priority-cell-{{locationId}}">
+                            <input
+                                class="ez-subitem-priority-input"
+                                id="priority-{{contentInfo.contentId}}"
```

Comment:
since the priority is a property of a Location, it would be a bit more logical to have an id based on the Location id instead.